orbmain/src/main/java/com/sun/corba/ee/spi/misc/ORBConstants.java

```diff
@@ -616,6 +618,10 @@ public static int makePersistent( int scid )
     // ORB's gmbal root.
     public static final String GMBAL_ROOT_PARENT_NAME = SUN_PREFIX 
         + "ORBGmbalRootParentName" ;
+    
+    //protocol constants to enable new lock api mechanism to process fragments 
+    public static final String ENABLING_NEW_FRAGMENT_CONCURRENCY_PROCESS = SUN_PREFIX + "protocol.enablingNewFragmentProcess";
```

Comment:
Final niggle - 'enable' makes more sense here than 'enabling'

```suggestion
    public static final String ENABLING_NEW_FRAGMENT_CONCURRENCY_PROCESS = SUN_PREFIX + "protocol.enableNewFragmentProcess";
```